Patches/Advanced Mechanoid Warfare/Patches/Patch_Ranged.xml

```diff
@@ -0,0 +1,886 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>[RWY]Advanced Mechanoid Warfare</li>
```

Comment:
All these ranged weapons also need their melee attacks patched. You can just copy the generic handgun/longgun/heavy weapon melee stats used.

Reply:
I think the reason the original mod had no melee stats for the weapons is because most of the mechs wielding guns have very strong melee.